ubuntudesign/cloud-vanilla-theme
# Port actions

# Done
- Added the social list styles as a component
- Updated the demo

## QA
- Run `gulp build` and check the demo
- Check the list-social part of the demo works ok
